src/models/GitHub.js

```diff
@@ -54,22 +57,27 @@ class GitHub extends Fetcher {
       date.setDate(date.getDate() - 31)
     }
     const dateStr = date.toISOString()
-    return this.get(`notifications?since=${encodeURIComponent(dateStr)}`,
-                    { fetchAll: true }).then(result => result.json)
+    const url = `notifications?since=${encodeURIComponent(dateStr)}`
+    return this.get(url).then(result => result.json)
   }
 
   // https://developer.github.com/v3/search/#search-issues
-  getTasks(query) {
-    const params = `?q=${encodeURIComponent(query)}`
-    const url = `${Config.githubApiUrl}/search/issues${params}`
+  getTasks(filter) {
+    const extraParams = filter.updatedAt ? ` updated:>${filter.updatedAt}` : ' is:open'
+    const params = `?q=${encodeURIComponent(filter.query + extraParams)}`
+    const url = `${Config.githubApiUrl}/search/issues${params}&per_page=100`
```

Comment:
So we'll only see the most recently updated 100 issues/PRs? I think this makes sense if the goal is to continuously be archiving/ignoring/snoozing tasks.